### DISCUSSION TITLE
iphone11 ios15.5 can't enable this plugin

### DISCUSSION THREAD

**Discussion by Andy-Curious** [remotely-save/remotely-save#154] [Q&A]
Failed to load plugin remotely-sav

![error](https://user-images.githubusercontent.com/15921129/179924353-564ba85e-872d-4844-85d6-dd1b77d403dd.jpg)

**Comment by fyears** [comment]
sorry but i don't have old device to test or debug, so no way to work around it. i suspect the reason is the web core is too old
